sdk/communication/azure-communication-phonenumbers/test/recordings/test_sip_routing_client_e2e.test_add_trunk.yaml

```diff
@@ -55,37 +55,37 @@ interactions:
       User-Agent:
       - azsdk-python-communication-phonenumbers/1.1.0b3 Python/3.10.0 (Windows-10-10.0.22621-SP0)
       x-ms-date:
-      - Tue, 15 Nov 2022 16:03:46 GMT
+      - Tue, 06 Dec 2022 10:31:18 GMT
       x-ms-return-client-request-id:
       - 'true'
     method: GET
     uri: https://sanitized.communication.azure.com/sip?api-version=2021-05-01-preview
   response:
     body:
-      string: '{"trunks":{"sbs1.sanitized.com":{"sipSignalingPort":1122},"sbs2.sanitized.com":{"sipSignalingPort":1123}},"routes":[]}'
+      string: '{"trunks":{"sbs3.3428b2faca6e4813885f4a3708457ed2.com":{"sipSignalingPort":2222}},"routes":[]}'
     headers:
       api-supported-versions:
       - 2021-05-01-preview, 2022-09-01-preview
       content-type:
       - application/json; charset=utf-8
       date:
-      - Tue, 15 Nov 2022 16:03:39 GMT
+      - Tue, 06 Dec 2022 10:31:05 GMT
       ms-cv:
-      - fYUkCgL5xUCI6TAHVxDEjA.0
+      - uLgPicC1KkqrpZm7yhA9UQ.0
       strict-transport-security:
       - max-age=2592000
       transfer-encoding:
       - chunked
       x-cache:
       - CONFIG_NOCACHE
       x-processing-time:
-      - 93ms
+      - 86ms
     status:
       code: 200
       message: OK
 - request:
     body: '{"trunks": {"sbs1.sanitized.com": {"sipSignalingPort": 1122}, "sbs2.sanitized.com":
-      {"sipSignalingPort": 1123}}}'
+      {"sipSignalingPort": 1123}, "sbs3.3428b2faca6e4813885f4a3708457ed2.com": null}}'
```

Comment:
It appears that this was not properly sanitized.

Reply:
Fixed sanitizer to cover these cases as well.